recipes/libdwarf/all/conanfile.py

```diff
@@ -56,7 +56,7 @@ def requirements(self):
             self.requires("libelf/0.8.13")
         self.requires("zlib/[>=1.2.11 <2]")
         if  Version(self.version) >= Version("0.9.0"):
-            self.requires("zstd/1.5.5")
+            self.requires("zstd/[^1.5]")
```

Comment:
```suggestion
            self.requires("zstd/[~1.5]")
```

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

The recommended version range for zstd is only patch version compatible.